src/preamble.js

```diff
@@ -221,7 +221,7 @@ function initRuntime() {
   runtimeInitialized = true;
 
 #if WASM_WORKERS
-  if (ENVIRONMENT_IS_WASM_WORKER) return __wasm_worker_initializeRuntime();
+  if (ENVIRONMENT_IS_WASM_WORKER) return wasmWorkerInitializeRuntime();
```

Comment:
I think reading this kind of change as well, something feels like is being lost here. The left side makes it obvious that this is invoking an internal mechanism in wasm workers, whereas this new code reads a bit like it might be a public API function that is also available to end users?

Reply:
The added an additional leading underscore to all these internal symbols.